Formula/p/python-identify.rb

```diff
@@ -0,0 +1,32 @@
+class PythonIdentify < Formula
```

Comment:
I definitely understand creating formula for either
1. build backends (to avoid unversioned build resources, like `flit-core`)
2. popular resources (like `six`)
3. resources with complicated build requirements (like `crytography`)

But since this is just a library only used twice, what's the benefit?

Reply:
In the long run, I'd like to build the ecosystem as we can? But definitely open to other ideas.

Reply:
cc @Homebrew/core for thoughts

Reply:
there are several things tied with this:
- better license chain management
- resource caching
- these are also aligned with other package managers

Reply:
The license chain is really not something I think Homebrew should get into. And if it's only used twice I doubt the caching is gonna make a big difference.

As for aligning with other package managers, I could not care less about that. Our packaging model has always been different from others.

Reply:
Also: other package managers don't have `resource`s and we have a shared download cache between resources. Personally I only really think:

> 3\. resources with complicated build requirements (like `crytography`)

are worth bottling separately as well as "resources which can be sensibly used as standalone applications".

Reply:
ok, points taken. I will follow these ideas when doing the python formulae bottling moving forward. 😄